src/Common/src/System/Net/HttpVersion.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace System.Net
 {
-    internal static class HttpVersion
+    public static class HttpVersion
```

Comment:
Presumably this file will move from being under src/Common to src/System.Net.Primitives in the next step?